packages/paypal-commerce-integration/src/components/PayPalCommercePaymentMethodComponent.tsx

```diff
@@ -75,8 +76,13 @@ const PayPalCommercePaymentMethodComponent: FunctionComponent<
                         paymentForm.submitForm();
                     },
                     onError: (error: Error) => {
+                        if (error.message === 'INSTRUMENT_DECLINED') {
```

Comment:
May I ask you to place this statement after `paymentForm.disableSubmit(method, true)`;

Reply:
moved